tools/interactive/interactivetool_rstudio.xml

```diff
@@ -1,11 +1,12 @@
-<tool id="interactive_tool_rstudio" tool_type="interactive" name="RStudio" version="0.1">
+<tool id="interactive_tool_rstudio" tool_type="interactive" name="RStudio" version="0.2">
```

Comment:
```suggestion
<tool id="interactive_tool_rstudio" tool_type="interactive" name="RStudio" version="0.2" profile="22.01">
```

Reply:
I'd remove the `detect_errors="aggressive"` ... if some user function writes to stderr that should probably not cause failed datasets.